bokehjs/src/lib/models/widgets/base_datetime_picker.ts

```diff
@@ -0,0 +1,106 @@
+import flatpickr from "flatpickr"
+
+import {BaseDatePicker, BaseDatePickerView} from "./base_date_picker"
+import {TimeLike} from "./time_picker"
+import * as p from "core/properties"
+
+export abstract class BaseDatetimePickerView extends BaseDatePickerView {
+  declare model: BaseDatetimePicker
+
+  override connect_signals(): void {
+    super.connect_signals()
+
+    const {
+      value, min_time, max_time, hour_increment,
+      minute_increment, second_increment, seconds, am_pm,
+    } = this.model.properties
+
+    this.connect(value.change, () => {
+      const {value} = this.model
+      if (value != null) {
+        this.picker.setDate(value)
+      } else {
+        this.picker.clear()
+      }
+    })
+    this.connect(min_time.change, () => this.picker.set("minTime", this.model.min_time))
+    this.connect(max_time.change, () => this.picker.set("maxTime", this.model.max_time))
+    this.connect(hour_increment.change, () => this.picker.set("hourIncrement", this.model.hour_increment))
+    this.connect(minute_increment.change, () => this.picker.set("minuteIncrement", this.model.minute_increment))
+    this.connect(second_increment.change, () => this._update_second_increment())
```

Comment:
In Panel we also have `enableSeconds`.

Reply:
The property is called just `seconds` (see below).